common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4830,6 +4842,12 @@
     },
     "sqlState" : "42K0E"
   },
+  "UNSUPPORTED_OUTPUT_MODE_FOR_STREAMING_OPERATION" : {
```

Comment:
All the touched error conditions are related to streaming output mode. Let's create the `STREAMING_OUTPUT_MODE` error condition with sub-classes: `UNSUPPORTED_DATASOURCE`, `INVALID` and  `UNSUPPORTED_OPERATION`. The error text of `STREAMING_OUTPUT_MODE` shall point out the invalid mode, and sub-classes should provide some details.

Reply:
@MaxGekk I updated the name of the error classes as suggested. Please take a look again when you have time.